chamber/utils/datastructures.py

```diff
@@ -20,11 +20,14 @@ def __getattr__(self, name):
 class Enum(AbstractEnum):
 
     def __init__(self, *items):
-        self.container = set(items)
+        self.container = OrderedDict(((item, item) for item in items))
```

Comment:
potřebuji aby při získávání atributu se mi vrátilo to co je v enumu nastavené. Např situace kdy hash objektů je stejný jsou equal ale objekt uvnitř enumu má v sobě něco navíc